src/controllers/movies/moviesrecommended.js

```diff
@@ -36,6 +36,9 @@ define(["events", "layoutManager", "inputManager", "userSettings", "libraryMenu"
                 showYear: true,
                 centerText: true
             });
+
+            // FIXME: Wait for all sections to load
```

Comment:
do you have any idea on how to fix it?

Reply:
1. Dirty way: add count of sections to load (like semaphore).
2. Make all `load*` functions to return `Promise` and wait for all in `loadSuggestionsTab`.
All above is not tested - only thoughts.